Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LauncherSamplePage.xaml.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using Uno.Gallery.ViewModels;
+using Windows.System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.NonUIFeatures, "Launcher", Description = "Provides the ability to launch websites or even apps.", DocumentationLink = "https://learn.microsoft.com/en-us/uwp/api/windows.system.launcher", DataType = typeof(LauncherSamplePageViewModel))]
+	public sealed partial class LauncherSamplePage : Page
+	{
+		public LauncherSamplePage()
+		{
+			this.InitializeComponent();
+		}
+
+		private async void LaunchSettingsButton_Click(object sender, RoutedEventArgs e)
+		{
+			var result = await Launcher.LaunchUriAsync(new Uri("ms-settings:sound"));
+
+			if (!result && ((Sample)DataContext).Data is LauncherSamplePageViewModel viewModel)
+			{
+				viewModel.IsSettingsInfoBarOpen = true;
+			}
+		}
+
+		private void LaunchWebsiteButtonAsync_Click(object sender, RoutedEventArgs e)
+		{
+			if (((Sample)DataContext).Data is LauncherSamplePageViewModel viewModel)
+			{
+				viewModel.LaunchUri();
+			}
+		}
+	}
+
+	public class LauncherSamplePageViewModel : ViewModelBase
+	{
+		public string SettingsButtonContent { get => GetProperty<string>(); set => SetProperty(value); }
+		public string LaunchUriText { get => GetProperty<string>(); set => SetProperty(value); }
+		public bool IsWebsiteInfoBarOpen { get => GetProperty<bool>(); set => SetProperty(value); }
+		public string WebsiteInfoBarText { get => GetProperty<string>(); set => SetProperty(value); }
+		public bool IsSettingsInfoBarOpen { get => GetProperty<bool>(); set => SetProperty(value); }
+
+		public LauncherSamplePageViewModel()
+		{
+#if __IOS__
+			//In case of iOS, any special URI opens the main page of system settings (there is no settings deep-linking available on iOS)
+			SettingsButtonContent = "Open Settings";
+#else
+			SettingsButtonContent = "Open Sound Settings";
+#endif
+			LaunchUriText = "https://platform.uno/";
+		}
+
+		public async void LaunchUri()
```

Comment:
```suggestion
		public void LaunchUri()
```

Reply:
Doesn't this suggestion make the `await Launcher.QueryUriSupportAsync` throw an error on all platforms except wasm?

Reply:
`async` should be kept
you can throw in a `await Task.CompletedTask` within `__WASM__` if the analyzers are complaining